README.md

```diff
@@ -118,27 +120,27 @@ Level of details in the logs, one of: `FATAL`, `ERROR`, `WARN`, `INFO`, `DEBUG`.
 
 ### `share-everyone`
 
-Automatically shares API collections created by the CI/CD task with everyone in your organization on the 42Crunch Platform. Accepted values are: `OFF`, `READ_ONLY`, `READ_WRITE`. Default is `OFF`. Note that the identity the action unders run (the owner of the API token) must have the `Share with Everyone` permission, otherwise the task will fail with a 403 error.
+Automatically shares API collections created by the CI/CD task with everyone in your organization on the 42Crunch platform. Accepted values are: `OFF`, `READ_ONLY`, `READ_WRITE`. Default is `OFF`. Note that the identity the action runs under (the owner of the API token) must have the `Share with Everyone` permission, otherwise the task will fail with a 403 error.
 
 ### `json-report`
 
-Writes audit execution report in JSON format to a specified file, optional. Default: `undefined`, no report is written. An [execution report](https://docs.42crunch.com/latest/content/tasks/integrate_github_actions.htm#scrollNav-4) details the list of APIs that where created, updated and deleted. This is useful if you want to automatically consume the results of the audit execution in a subsequent pipeline step .
+Writes an audit execution report in JSON format to the specified file. An [execution report](https://docs.42crunch.com/latest/content/tasks/integrate_github_actions.htm#scrollNav-4) details the list of APIs that were created, updated and deleted. This is useful if you want to automatically consume the results of the audit execution in a subsequent pipeline step. By default, no report is written.
 
 ### `api-tags`
 
 The CI/CD task can automatically assign tags to newly created APIs. Tags are specified in the following format: `category1:name1 category2:name2`. This flag is *optional*.
 
 ### `sarif-report`
 
-Converts the audit raw JSON format to SARIF and save the results into a specified file. By default, no report is written.
+Converts the audit raw JSON format to SARIF and save the results into the specified file. By default, no report is written.
 
 ### `audit-timeout`
 
-Sets the maximum timeout (in seconds) for the audit report. Fail if the result isn't ready within that interval. Default: `600`
+Sets the maximum timeout (in seconds) for the audit report. The task will fail if the result isn't ready within that interval. Default: `600`
 
 ## Prerequisites
 
-Create an API token in 42Crunch Platform and copy its value into a [repository secret](https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets) named `API_TOKEN`.
+Create an API token on the 42Crunch Platform and copy its value into a [repository secret](https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets) named `API_TOKEN`.
```

Comment:
The original "in" is the correct one to use here. Proper names don't take articles ("the") unless the article is part of the proper noun ("The Bible"), and "in" is more logical as you are doing things within the confines of the product.